src/DataFixtures/ORM/LoadCitizenProjectData.php

```diff
@@ -332,6 +332,7 @@ public function getDependencies()
     {
         return [
             LoadAdherentData::class,
```

Comment:
`LoadAdherentData` is no longer required here, because `LoadCommitteeData` depends on `LoadAdherentData`